text/0061-error-format-and-definitions.md

```diff
@@ -728,6 +728,54 @@ HTTP Code: `404 Not Found`
 
 ---
 
+## invalid_task_status
+
+### Context
+
+This error happens when a requested task status is invalid.
+
+#### Error Definition
+
+HTTP Code: `400 Bad Request`
+
+```json
+{
+    "message": "`:status` is invalid. Available task statuses are: `:taskStatuses`.",
+    "code": "invalid_task_status",
+    "type": "invalid_request",
+    "link":"https://docs.meilisearch.com/errors#invalid_task_status"
+}
+```
+
+- The `:status` is inferred when the message is generated.
+- The `:taskStatuses` is inferred when the message is generated.
```

Comment:
I am not sure what that means

Reply:
This is just to say that the markers starting `:` in the error message are actually displayed at the time the error is returned by replacing by the real value of `:status` and `:taskStatuses`.

This is specified on all our error messages, I don't think there's any point in specifying it, maybe it's understandable when reading the error message.